docker/prod/prod.template.yaml

```diff
@@ -121,6 +124,29 @@ spec:
             requests:
               cpu: 50m
               memory: 50Mi
+        - name: exporter
+          image: artifactory.wikia-inc.com/platform/nginx-exporter-prod:${IMAGE_TAG}
+          livenessProbe:
+            httpGet:
+              path: /metrics
```

Comment:
that's a hack ;-)

Reply:
Yes but It beats adding code to the app ;)